src/Api.js

```diff
@@ -80,17 +76,26 @@ export default class Api extends ToolApi {
   * @param {Boolean} warnOnError - if true, then will show message on selection change
   * @return {Function}
   */
-  validateVerse(targetVerse, skipCurrent = false, contextId = null, warnOnError = false) {
+  validateVerse(targetVerse, skipCurrent = false, contextId = null, ) {
     let {
       tc: {
         username,
         changeSelections,
```

Comment:
this should be nested under `tc.actions`. And you'll need to update where this is added in tC when selecting a project.
This is because when the tool is opened this action along with all the others are put into `tc.actions`. As is, this code will only work before a tool is selected.